ONSdigital/sdc-int-rh-ui
# 611 Fix left over references to old config

# Motivation and Context

#79 Fixed the config to default to prod values and removed `APP_SETTINGS`, but there were some leftover references it missed. Also document the unmaintained make targets a bit and remove some broken tasks, reduce unnecessary use of tasks.

# What has changed



- Removed remaining APP_SETTINGS references
- Fix lots of little things 

# How to test?



No functional change, check the make targets we use still work, tests still pass etc.

# Links



https://trello.com/c/c9KEbaWL/611-tidy-up-rh-config
#79
